src/celeritas/em/interactor/MuBremsstrahlungInteractor.hh

```diff
@@ -177,14 +193,12 @@ CELER_FUNCTION real_type MuBremsstrahlungInteractor::sample_cos_theta(
 
 //---------------------------------------------------------------------------//
 
-CELER_FUNCTION real_type MuBremsstrahlungInteractor::differential_cross_section(
-    real_type gamma_energy) const
+CELER_FUNCTION real_type
+MuBremsstrahlungInteractor::calc_differential_xs(real_type gamma_energy) const
```

Comment:
This should probably be moved to a standalone helper class since we will likely need it one day for generating our own cross sections. It would also be good for testing...

Reply:
Already started this, actually!